framework/src/application/node/forger/index.js

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright © 2019 Lisk Foundation
+ * Copyright © 2020 Lisk Foundation
```

Comment:
I update those for every file which is changed in PRs.

Reply:
I think we should not do that.. we should only use `2020` for new files because we want the license to be associated with old code as well.. so if we change the license implication can be different.

Reply:
But any source code which is changed in 2020, should have updated license header.